cudf/tests/test_dataframe.py

```diff
@@ -356,6 +356,34 @@ def test_dataframe_to_string_wide():
     assert got.split() == expect.split()
 
 
+def test_dataframe_empty_to_string():
+    # Test for printing empty dataframe
+    df = DataFrame()
+    got = df.to_string()
+    print(got)
+    expect = '''
+Empty DataFrame
+Columns: []
+Index: []
+'''
+    # values should match despite whitespace difference
+    assert got.split() == expect.split()
+
+
+def test_dataframe_emptycolumns_to_string():
+    # Test for printing dataframe having empty columns
+    df = DataFrame({'a': [], 'b': []})
+    got = df.to_string()
+    print(got)
+    expect = '''
```

Comment:
Please correct indentation.
```suggestion
    expect = "Empty DataFrame\nColumns: ['a', 'b']\nIndex: []\n"
```

Reply:
corrected indentation.